ledger/appcow.go

```diff
@@ -425,6 +430,18 @@ func (cb *roundCowState) DelKey(addr basics.Address, aidx basics.AppIndex, globa
 	return nil // note: deletion cannot cause us to violate maxCount
 }
 
+// AppendLog creates a new key-value {addr, aidx} in log
+func (cb *roundCowState) AppendLog(aidx basics.AppIndex, value basics.TealValue) error {
```

Comment:
Earlier I suggested "AddLog". Given this naming, "AppendLog" could be used to match.